### PR TITLE
Local Github ACTions 

### PR DESCRIPTION
**Pull request by D-DePablos**
## Changes:
- Added ACT environment variables
- Investigating Docs deployment error

## Types of changes

What types of changes does your code introduce?
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes

Testing: Local runs are done with this .actrc from an M1 macbook:

-P ubuntu-latest=ghcr.io/catthehacker/ubuntu:act-latest
-P ubuntu-18.04=ghcr.io/catthehacker/ubuntu:act-18.04
-b
--container-architecture linux/amd64

### CODE REVIEW

**Comment by D-DePablos** [comment]
TODO: I think this branch is bugged from changes to one or multiple actions.

**Comment by D-DePablos** [comment]
This is giving me some problems due to how ACT creates an ACT.env parameter that is not available on GitHub. Actions seem to update when updating the PR, so they were not working for a while. They should be changed with care.

**Review by stdevMac — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -44,11 +43,9 @@ codecov-test:
 	courtney/courtney -v -o coverage/coverage.out ./...
 	@cd internal/db && $(MAKE) rm-notest
 
-tidy: ## add missing and remove unused modules
-	 go mod tidy
-
 format: ## run go formatter
 	gofumpt -l -w .
+	go mod tidy
```
Comment (joshklop) [inline Makefile:48]: I think implicitly running `go mod tidy` on the `format` command is unnecessary, and obfuscates what the actual command is doing. I personally prefer having `tidy` and `format` separated.

**Review by joshklop — CHANGES_REQUESTED** [review]
Are there more changes with the GitHub actions? I only am seeing changes to the Makefile, go.mod, and go.sum.

**Comment by D-DePablos** [comment]
This is quite a bit of a larger headache than I had thought. Additionally, if there is a correct docs staging environment, and smoke tests for Juno, it is not as useful.

The main pain points are related to how in ACT you must use a Docker container, while in GH actions you have a full system. ACT proves to be slower, less reliable, and to force some if conditionals on the workflow files.

**Pull request closed without merging**